include/MainWindow.h

```diff
@@ -263,6 +264,13 @@ class MainWindow : public QMainWindow
          */
         unsigned int getSelectedRulesCount();
 
+        /**
+         * @brief Ask user to proceed with closing old file due to openning of new one
+         *
+         * @return user answer
+         */
+        QMessageBox::StandardButton openNewFileQuestionDialog(const QString &oldFilepath);
```

Comment:
The SCAP Workbench codebase uses the `const T& var` convention instead of `const T &var`.